<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseSearchDialog</class>
 <widget class="QDialog" name="ChooseSearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="layout_combo"/>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="edit_query"/>
   </item>
   <item>
    <widget class="QScrollArea" name="area_parameters">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>380</width>
        <height>68</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_search">
     <property name="text">
      <string>Search!</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
